jobs/e2e_node/crio/latest/image-config-cgroupv1-hugepages.yaml

```diff
@@ -5,6 +5,6 @@ images:
   ubuntu:
     image_family: fedora-coreos-stable
     project: fedora-coreos-cloud
-    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupsv2_hugepages.ign"
+    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupv1_hugepages.ign"
```

Comment:
Is this right?

If so, we should have a groups v2 huge pages job and you should change the file name.

Reply:
I assume that's a git thing how to handle the diff. We still have `image-config-cgroupv2-hugepages.yaml` which refers the v2 config.

Reply:
I see. That is what I get for reviewing on my phone.